docs/ubuntu.md

```diff
@@ -74,6 +73,11 @@ you will need to run again this last line.*
     wget https://raw.githubusercontent.com/umap-project/umap/master/umap/settings/local.py.sample -O /etc/umap/umap.conf
 
 
+## Make umap use the local configuration file
+
+   export UMAP_SETTINGS=/srv/umap/local.py
```

Comment:
Humm, the tutorial should use the default configuration path, which is `/etc/umap/umap.conf`, and should be loaded by uMap without needing an env var. Have you tested this way?

Reply:
I just tested this and uMap doesn't seem to be using umap.conf. That is probably why after searching I used local.py with the environment settings.

Reply:
Looking at the commits and the releases, the commit https://github.com/umap-project/umap/commit/4c8b6f79de7b49692ed6a2b75c25b7e5ea6d4dfb for umap.conf is after the last release 0.8.2. Maybe that would explain it?

Reply:
Just had a look at settings/__init__.py and that contains the old code. I replaced it with the new version and now it works, so it is a version mismatch.